mvlearn/embed/kcca.py

```diff
@@ -167,266 +218,60 @@ def fit_transform(self, Xs):
             The data for kcca to fit to.
             Each sample will receive its own embedding.
 
-        Returns
-        -------
         weights_ : list of array-likes
                    Canonical weights
```

Comment:
I think these are backwards. If this inherits from BaseEmbed, then fit() should return self, and transform should return Xs_transformed. https://github.com/neurodata/mvlearn/blob/master/mvlearn/embed/base.py

That way, fit_transform can just call .fit().transform() and return Xs_transformed